src/compiler/checker.ts

```diff
@@ -222,11 +222,10 @@ namespace ts {
             getSuggestionForNonexistentProperty,
             getSuggestionForNonexistentSymbol,
             getBaseConstraintOfType,
-            getJsxNamespace,
-            resolveNameAtLocation(location: Node, name: string, meaning: SymbolFlags): Symbol | undefined {
-                location = getParseTreeNode(location);
-                return resolveName(location, name, meaning, /*nameNotFoundMessage*/ undefined, name);
+            resolveName(name, location, meaning) {

```

Comment:
That said, `resolveNameAtLocation` (nee. `resolveName`) is marked `/* @internal */` and isn't used in transforms. The guard is mostly present in case we decide to use it at some point down the road.